metadataStore/odm_templates.py

```diff
@@ -113,24 +146,23 @@ class Event(Document):
 
     Attributes
     ----------
-    descriptor : metadataStore.odm_templates.EventDescriptor
-        Foreign key to EventDescriptor that provides info regarding a
-        set of events of the same type
-
-    data : dict
-        Data dictionary where experimental data is stored.
-
-    time : timestamp
-        The date/time as found at the client side when an event is
-        created.
+    descriptor : mongoengine.DynamicDocument
+        EventDescriptor foreignkey
```

Comment:
It is a foreign key to another table. EventDescriptor has its own declaration of purpose and attribute documentation. Database doc template